src/imageio/imageio_rawspeed.cc

```diff
@@ -529,6 +528,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed_sraw(dt_image_t *img, RawImage r, d
           for(int k = 0; k < 3; k++)
           {
             out[k] = (float)in(j, cpp * i + k) / (float)UINT16_MAX;
+            out[3] = 0.0f;
```

Comment:
I do not get why you put loop-independent constant assignment into a nested for-loop, also in line 551. Can't you just move those lines below the lines where ``out`` is defined?

Reply:
just stupid me. Thanks!

Reply:
ah, okay, I thought I was stupid too, not getting the big picture :D